requirements.txt

```diff
@@ -5,3 +5,5 @@
 # See: https://caremad.io/2013/07/setup-vs-requirement/
 
 --editable .
+
+requests
```

Comment:
since requests is already on setup.py there is no need to modify the requirements.